.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,22 @@
+name: Shellcheck scope.sh
+
+on: 
+  push:
+    paths:
+      - '.github/workflows/shellcheck.yml'
+      - 'ranger/data/scope.sh'
```

Comment:
Let's prepend a slash to this path, so that we only match `ranger/data/scope.sh` and not `anything/ranger/data/scope.sh`. Mostly for the sake of correctness.

Reply:
That is, assuming it works like gitignore. If it doesn't, ignore my comment. :)

Reply:
So, the GH Actions seem to use [this function](https://golang.org/pkg/path/#Match) for matching.

From its docs:
  > Match requires pattern to match all of name, not just a substring.

If I read this right that means it won't match anything but `ranger/data/scope.sh` so I think prefixing `/` wouldn't work anymore.

I just force-pushed a test to verify that nothing gets triggered, neither `ghcactions/ranger/data/scope.sh` nor `ghcactions-ranger/data/scope.sh` triggered the shellcheck action.